activerecord/test/cases/relation/delete_all_test.rb

```diff
@@ -12,18 +12,37 @@ class DeleteAllTest < ActiveRecord::TestCase
   def test_destroy_all
     davids = Author.where(name: "David")
 
+    assert_difference("Author.count", -1) do
+      davids.destroy_all
+    end
+  end
+
+  def test_destroy_all_returns_nil
```

Comment:
How about?
```suggestion
  def test_destroy_all_no_longer_returns_a_collection
```
I'm not 100% happy with this naming but I want to remark that the change here is that the collection is no longer return as opposed to the next example.